README.md

```diff
@@ -133,27 +130,22 @@ Run `dot --help` to get a full list of available options.
 
     ```bash
     dot \
-    --swap_type simswap \
+    -c ./configs/simswap_hq.yaml \
     --target 0 \
     --source "./data" \
-    --parsing_model_path ./saved_models/simswap/parsing_model/checkpoint/79999_iter.pth \
-    --arcface_model_path ./saved_models/simswap/arcface_model/arcface_checkpoint.tar \
-    --checkpoints_dir ./saved_models/simswap/checkpoints \
-    --crop_size 512 \
     --show_fps \
     --use_gpu
     ```
 
-    Additionally, to enable face superresolution, use the flag `--gpen_type gpen_256` or `--gpen_type gpen_512`.
+    Additionally, to enable face superresolution, change the configuration file to `configs/simswaphq_gpen_256.yaml` or `configs/simswaphq_gpen_512.yaml`.

```

Comment:
Keep the previous gpen documentation: 

```
use the flag --gpen_type gpen_256 or --gpen_type gpen_512
```

Reply:
sure.

btw, all the existing flags are still valid. It's only when you pass a config file path, that certain flags are overriden by the loaded config.

Reply:
[c89cf77](https://github.com/sensity-ai/dot/pull/17/commits/c89cf77337115f89394187a20e154e63540b7400)